brave/gulpfile.js/brave-replace-paths.js

```diff
@@ -156,7 +156,7 @@ const createBraveReplacePathsTask = () => {
       )
       .pipe(
         replace(
-          /'(.*)\/restore-vault'/gm,
+          /'(.*)\/keychains\/restore-vault'/gm,
```

Comment:
This regex was too general, it would override the route constant on: https://github.com/brave/ethereum-remote-client/blob/master/ui/app/helpers/constants/routes.js#L19